### DISCUSSION TITLE
Hottop and USB-C Cables

### DISCUSSION THREAD

**Discussion by FilmScoreMonger** [artisan-roaster-scope/artisan#1079] [Q&A]
I have a Hottop coffee roaster and am controlling it with Artisan. I have been doing so for years. However, the cable I've been using the entire time I believe needs to be replaced — I have to hook it through a USB-C adapter for my newer Macbook, and I'm having the roaster intermittently shut off and back on again during roasts. (Yikes!)

So I purchased a USB C to USB A 2.0 cable, hoping to be able to use that instead. I've now tried several cables and none of them are being recognized by Artisan via Config > Serial Port.

Am I missing a step here? How do I get Artisan to recognize this cable? :)

**Comment by MAKOMO** [comment]
There have been reports that newer macOS version do not play well with non-apple USB-C-2-USB-A adapters.

**Reply by FilmScoreMonger** [reply]
That doesn't surprise me. But that's why I went for a USB-C > USB-A cable ... I've used other cables for other applications with my Mac, but this is the first one that's given me this much grief.

**Comment by CarefreeBuzzBuzz** [comment]
You need to make sure that the cable is not nearly for charging, but it’s also for data transmission. With a proper cable, I have had no issues with a USB-C to USB-A adapter.  Please go back and look at the specs for the cables you purchased. If they supposedly transmit data, it could be that Hottop use some sort of proprietary cable.

**Reply by FilmScoreMonger** [reply]
It's the proper cable according to the manufacturer. I've reached out to Hottop in the meantime. Thank you!
